www/http/validator_test.go

```diff
@@ -63,4 +64,45 @@ func TestValidator(t *testing.T) {
 		assert.Equal(t, w.Code, 400)
 		fmt.Println(w.Body)
 	})
+
+	t.Run("Shall return a validator", func(t *testing.T) {
```

Comment:
Let's move these tests to a new function: `TestValidatorByNumber`

Also, try to cover the situation that validator can't be found, like number is `val.Number()+1`